ecosystem/indexer-server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16 as builder
+ARG PATH
+
+WORKDIR /usr
+
+RUN echo "copy indexer server..."
+COPY . .
+
+WORKDIR /usr/typescript
+
+RUN echo "install and generate the server..."
+RUN yarn install
+RUN yarn run generate
+
+EXPOSE 4000
+CMD ["yarn", "run", "dev"]
```

Comment:
is there a non-dev way to run the server?

Reply:
@hariria  ^. I am basically following Andrew's README

Reply:
I think there is no dev to run the server now

Reply:
what about `ts-node --transpile-only`?

Reply:
I'll stamp for now to get something out fast, but recommend you guys taking another look